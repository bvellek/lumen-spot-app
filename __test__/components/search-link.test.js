import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import SearchLink from '../../public/js/components/link-section';

describe('<SearchLink />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <SearchLink linkTitle='test title' accessibilitySpan='test context accessibility info' />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
