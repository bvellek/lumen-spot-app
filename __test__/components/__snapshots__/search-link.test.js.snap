// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchLink /> it renders correctly 1`] = `
<section
  className="link-section"
>
  <h2>
    <SearchLink
      linkTitle="Find Your Light"
    />
  </h2>
</section>
`;
